src/kayenta/edit/configDetailHeader.tsx

```diff
@@ -1,40 +1,87 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
+import { UISref } from '@uirouter/react';
 
 import { ICanaryState } from 'kayenta/reducers';
-import { ICanaryConfig } from 'kayenta/domain/ICanaryConfig';
-import ConfigDetailActionButtons from './configDetailActionButtons';
+import { ICanaryConfig } from 'kayenta/domain';
 import { mapStateToConfig } from 'kayenta/service/canaryConfig.service';
 import FormattedDate from 'kayenta/layout/formattedDate';
 
+import ConfigDetailActionButtons from './configDetailActionButtons';
+
 interface IConfigDetailStateProps {
   selectedConfig: ICanaryConfig;
+  editingDisabled: boolean;
 }
 
+const getOwnerAppLinks = (owners: string[]) => {
+  if (owners.length === 1) {
+    return (
+      <UISref to="." params={{ application: owners[0] }}>
+        <a>{owners[0]}</a>
+      </UISref>
+    );
+  } else {
+    // totally gross to read, but a somewhat-straightforward way of creating
+    // a 'one, two, or three' sentence from this array of app names with some JSX in between
```

Comment:
Maybe we get fancy and make this an importable util method for other areas. I'll keep it in mind for future designs.